Book: Added small section on primary cred fallback (#3365)
kanidm/kanidm: book/src/accounts/account_policy.md

kanidm group account-policy webauthn-attestation-ca-list idm_all_persons trusted-authenticators
```

### Setting Primary Credential Fallback

The primary credential fallback enables behavior which allows authenticating
using the primary account password when logging in via LDAP.

> If both a LDAP and primary password are specified it will only accept the LDAP password.

Reviewer: ```suggestion
If both an LDAP and primary password are specified, Kanidm will only accept the LDAP password.
```

> ```bash

Reviewer: Is there "disable" option? How does one complete that step?

Author: Default is set to false. If its enabled and it should be disabled, just running:
`kanidm group account-policy allow-primary-cred-fallback <group name> false` will work.

Author: No stress, can do

Author: @yaleman sorry for the delay. (Got very busy with work). This good?

Reviewer: No worries, I'm on holiday and this is a fun thing not super urgent 😃 thanks for the update
